download.html

```diff
@@ -23,6 +23,8 @@
   <p>TripleA&nbsp;for Linux</p>
 </a>
 
+<p>Missing a version? Checkout <a href="https://github.com/triplea-game/triplea/releases/latest">all versions here</a>.</p>
```

Comment:
(1) The wording maybe could be different. Not sure if everyone would think of version as say "v1.9" vs "v1.8". I'd re-word this to "Download for other platforms"

(2) Can we still show a download page? Maybe copy/paste the old version of the DL page and call it the 'all-platform' page. It's intended to avoid users from looking at the github release list, it's overwhelming for some.

Reply:
1. Agreed
2. Would an "un-hide" button be enough? So that all the remaining installers can be shown if requested.

Reply:
A "show installers for all platforms" button could work too. That would be similar to the link going to a different page. Offhand I think it's better to show the options off the bat with the right one up top and center. Up to you at this point IMO, main requirement I think is that the other installers not be from github releases.